.github/workflows/ci-build.yml

```diff
@@ -23,12 +36,17 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
+        cache: pip
+    - name: Install compatible cryptography
+      if: matrix.python-version == 'pypy3.7' || matrix.python-version == 'pypy3.6'
```

Comment:
We'd like to avoid making the build settings complex just to support old pypy versions. Can you remove pypy 3.6 to 3.9? I don't believe this SDK can cause issues only for old pypy runtimes as this SDK mainly does web API calls, parsing HTTP request data, and database access.